src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequencesHeader.tsx

```diff
@@ -66,10 +66,10 @@ const BlastInputSequencesHeader = (props: Props) => {
   };
 
   const scrollToLastInputBox = () => {
-    const lastTextarea = document.querySelector(
-      `.${sequenceBoxStyles.inputSequenceBox}:last-child textarea`
-    ) as HTMLTextAreaElement;
-    lastTextarea?.focus();
+    const lastInputBox = document.querySelector(
+      `.${sequenceBoxStyles.inputSequenceBox}:last-child`
+    ) as HTMLDivElement;
+    lastInputBox.scrollIntoView({ block: 'end', behavior: 'smooth' }); // Safari doesn't properly support this (see https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView); but other options scroll just as jerkily on Safari
```

Comment:
will this not work fine `lastInputBox.scrollIntoView()` since we have the scroll smooth behavior define in the css.
https://github.com/Ensembl/ensembl-client/blob/empty-blast-input-boxes/src/content/app/tools/blast/views/blast-form/BlastForm.scss#L26

Reply:
This is the scrolling behaviour in Chrome:

https://user-images.githubusercontent.com/6834224/156369560-4bc0829a-ffa3-40b4-8c35-c1da6de1b435.mp4

And this in Safari:

https://user-images.githubusercontent.com/6834224/156369637-b0100ced-8d28-43c3-a844-62f9891676c6.mp4

Surprisingly, the instant scrolling is Safari is not fixed by the `scroll-behaviour: smooth` CSS rule on the container. Meaning that Safari instantly scroll to the box even if we use the focus on the textarea method.